js/cm6.mjs

```diff
@@ -1,4 +1,4 @@
-export { EditorState, basicSetup } from "@codemirror/basic-setup";
-export { EditorView } from "@codemirror/view";
-export { oneDark } from "@codemirror/theme-one-dark";
-export { javascriptLanguage } from "@codemirror/lang-javascript";
+export { EditorState, basicSetup } from "@codemirror/basic-setup";
+export { EditorView } from "@codemirror/view";
+export { oneDark } from "@codemirror/theme-one-dark";
+export { javascriptLanguage } from "@codemirror/lang-javascript";

```

Comment:
What's the diff in this file? :thinking:

Reply:
It seems that their newlines are inexplicably changed, I'll restore them later.😄